### ISSUE TITLE
Other PubChem-based tools

### ISSUE THREAD

**Issue by DeniseSl22** [jacobwindsor/pubchem-ranker#10]
Hi Jacob,

Upon searching for other PubChem tools, I found: https://ccj.springeropen.com/articles/10.1186/1752-153X-2-11

It does not have the same features as the ranker program (thank god), however I like the layout and applications that is has... Since it can only be used on Windows XP or Vista (*sigh*), perhaps we can re-use some of their ideas ;)?

**Comment by jacobwindsor** [comment]
Nice find!

The downloading of the files is the most useful I guess. Including the SMILEs of the compound would be a nice addon too :)

**Comment by DeniseSl22** [comment]
The second tool I found using PubChem as a databse:
https://academic.oup.com/bioinformatics/article/28/1/140/221571/PubChem-promiscuity-a-web-resource-for-gathering

**Comment by DeniseSl22** [comment]
?Just added the second article I found about PubChem tools.

________________________________

**Comment by jacobwindsor** [comment]
Ah, forgot to reply yesterday. This is much more similar the PubChem ranker but even a little simpler. I'm a but surprised they published tbh

**Comment by DeniseSl22** [comment]
Yeah, maybe we should show this to Egon... If it is too much alike, we should either improve the Ranker Program in order to publish about it, or not publish at all... I have a meeting with him (and Chris) upcoming Tuesday... What do you think? Can we improve the Ranker Program, so that it is far more superior than the promiscuity tool?

**Comment by jacobwindsor** [comment]
Yep, we should do that. The PubChem ranker is already slightly better since it can do both bio-assays and bio-pathways.

The downfall of the promiscuity app, IMO, is that it focuses just on promiscuity which is rather a niche. The goal of the PubChem ranker is somewhat broader - to rank a list of compounds by the amount of available data. Rather than investigating drug potential, it is investigating any potential. For that goal then it makes sense to include other databases.

For the VOCs project, it was useful since it gave an indication of which compounds were worth researching. We didn't have a lot of time so wasting it searching for compounds with virtually no data wasn't an option. 

BTW: If this becomes more than a PubChem tool we should probably think of another name for it.

**Comment by DeniseSl22** [comment]
Yeah that was what I was thinking.... However, since they got their tool published, it is more probable that we will succeed as well in this.

Okay, I will talk to Egon about this next week.

New name (just a small brainstorm): Data Availability Ranker tool (DAR-tool?)

**Comment by DeniseSl22** [comment]
http://regexr.com/ for advanced users...
